### DISCUSSION TITLE
Publish gh-pages to non-origin repo?

### DISCUSSION THREAD

**Discussion by ihnorton** [quarto-dev/quarto-cli#1243] [Q&A]
Is it possible to target a repo other than the `origin` with `quarto publish gh-pages`? I read through the [template](https://github.com/quarto-dev/quarto-cli/blob/046f7fc0228e4480b4d9af57f92b5538f8a27e5b/src/publish/gh-pages/gh-pages.ts) and it appears not, _unless_ I misunderstood something about the handling of `siteUrl`. If not, please consider this a feature request 😄 Thanks!

**Comment by adrien-perello** [comment]
I think you might want to check the discussion: [Is it possible to build a website from multiple repos?](https://github.com/quarto-dev/quarto-cli/discussions/1282)
